### ISSUE TITLE
train, validation, test split and hyperparameter tuning

### ISSUE THREAD

**Issue by BernhardAhrens** [EarthyScience/EasyHybrid.jl#57]
We should explore how we want to do hyperparameter tuning (such as batchsize, optimiser, learning rate, weight decay).

I am aware of 

- https://github.com/baggepinnen/Hyperopt.jl 
- https://github.com/ThummeTo/DistributedHyperOpt.jl
- https://github.com/JuliaAI/MLJTuning.jl

Not sure how well all of there play with Lux.jl or if there is a "Lux" way for parameter optimization.

Maybe just custom loops would work best. For this we then also need split in train, validation, and test
